runtime/centrifuge/src/lib.rs

```diff
@@ -1704,6 +1708,40 @@ impl pallet_membership::Config for Runtime {
 	type WeightInfo = pallet_membership::weights::SubstrateWeight<Self>;
 }
 
+parameter_types! {
+	#[derive(Clone, PartialEq, Eq, Debug, TypeInfo, Encode, Decode, MaxEncodedLen)]
+	pub const MaxFeedersPerKey: u32 = 10;
+	pub const FirstValueFee: Fee = Fee::Balance(deposit(1, pallet_oracle_feed::util::size_of_feed::<Runtime>()));
```

Comment:
This means `1 CFG` per value?

Reply:
`1.65 CFG` for each first value, yes. See here for context: https://github.com/centrifuge/centrifuge-chain/pull/1629#discussion_r1432380994